src/OVAL/probes/probe-api.c

```diff
@@ -1794,4 +1794,52 @@ SEXP_t *probe_obj_getmask(SEXP_t *obj)
     SEXP_free(objents);
     return (mask);
 }
+
+static bool path_startswith(const char *path, const char *prefix)
```

Comment:
I do feel like having this function in `common/util.c` instead. We have a group of path-related functions there already.

Also, unlike counterparts from util.c, this one does not account for Windows environment. Is this an intended limitation?